types/shared.d.ts

```diff
@@ -1,24 +1,36 @@
-declare module "skedify-sdk" {
+declare module 'skedify-sdk' {
   export interface APIError<Entity> {
-    code: string;
-    message: string;
-    ticket_id?: string;
+    code: string
+    message: string
+    ticket_id?: string
     fields?: {
-      [Key in keyof Entity]?: string[];
-    };
+      [Key in keyof Entity]?: string[]
+    }
   }
 
   export interface APIResponse<Entity> {
-    data: Entity;
-    warnings: unknown[];
-    errors: APIError<Entity>[];
-    status: number;
+    data: Entity
+    warnings: unknown[]
+    errors: APIError<Entity>[]
+    status: number
+    paging: PagingResponse
+  }
+
+  export interface PagingResponse {
+    current: number
+    from: number
+    hasNext: () => boolean
+    hasPrevious: () => boolean
+    pages: number
+    size: number
+    to: number
+    total: number
```

Comment:
For me personally some of these names aren't that descriptive.

`current` -> `currentPage` | `page`
`pages` -> `totalPages` 
`total` -> `totalResults`

But this is more of a personal prefence

Reply:
@Jarrku I didn't change these, but they came from the docs from the SDK. (see https://github.com/skedify/sdk-js#paging)
I agree, will change them on the SDK and here